### ISSUE TITLE
No Progress Bar When Uploading songs Via Music Files For Screen-Reader Users

### ISSUE THREAD

**Issue by theblindadvocacyradio** [AzuraCast/AzuraCast#5383]
### Installation Method

One-Click Installation (Docker)

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #49b3191

### What happened?

When I upload songs via the music files area on AzuraCast, my screen-reader doesn't tell me the progress of the upload. I am using NVDA (Non-Visual Desktop Access). There should be a way for screen-readers to announce the percentage of a progress bar of songs that are uploading.

**Comment by SlvrEagle23** [comment]
@theblindadvocacyradio We actually already did display a progress bar during uploads, it just didn't have the proper ARIA attributes to be seen by screen readers as a progress bar. I've updated the progress bar display to properly set the accessible values as it updates.
